Makefile

```diff
@@ -350,43 +350,51 @@ helm-chart-generate: kustomize helm kubectl-slice yq charts
 
 
 .PHONY: helm-package
-helm-package: cr helm-chart
-	@echo "== CHART RELEASER (package) =="
-	$(CR) package ./charts/awx-operator
+helm-package: helm-chart
+	@echo "== Package Current Chart Version =="
+	mkdir -p .cr-release-packages
+	# package the chart and put it in .cr-release-packages dir
+	$(HELM) package ./charts/awx-operator -d .cr-release-packages
 
 # List all tags oldest to newest.
 TAGS := $(shell git ls-remote --tags --sort=version:refname --refs -q | cut -d/ -f3)
 
-# The actual release happens in ansible/helm-release.yml
-# until https://github.com/helm/chart-releaser/issues/122 happens
+# The actual release happens in ansible/helm-release.yml, which calls this targer
+# until https://github.com/helm/chart-releaser/issues/122 happens, chart-releaser is not ideal for a chart
+# that is contained within a larger repo, where a tag may not require a new chart version
 .PHONY: helm-index
-helm-index: cr helm-chart
-	@echo "== CHART RELEASER (httpsorigin) =="
-	git remote add httpsorigin "https://github.com/$(CHART_OWNER)/$(CHART_REPO).git"
-	git fetch httpsorigin
+helm-index:
+	@echo "== GENERATE INDEX FILE =="
+	# checkout gh-pages branch and push up the new index file
+	git switch gh-pages
+	# TODO: Assess removal of additional origin
+	git fetch --all --tags
```

Comment:
With `--all`, all remotes will be fetched. Is that intended?  Do you maybe want `git fetch origin --tags`?

Reply:
Yeah I think that would be fine, i was having trouble getting all of the tags on my fork, I can remove --all as well as the TODO